<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1750</width>
    <height>897</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1750</width>
    <height>900</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>ECSLOGCHK</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>300</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QTextEdit" name="textEditFind">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSearchNo">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>검색 수</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="LCurrNo">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="BtnFind">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>찾기</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCurrNo">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>현재 번호</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="LSearchNo">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="BtnCurrRowSel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>현재 열 선택</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QComboBox" name="comboBox">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>메세지 ID 리스트</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CONVEYOR_EVENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CONVEYOR_REPLY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SEND_EVENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SEND_REPLY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TRANSFER_IN_ECS_EVENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TRANSFER_IN_ECS_REPLY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TRANSFER_OUT_ECS_EVENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TRANSFER_OUT_ECS_REPLY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ALARM_EVENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATE_EVENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DATE_REPLY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TRAY_EVENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TRAYID_MATCHING_EVENT</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="labelSearchNo_2">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>로그 범위 설정</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>10%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>70%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>80%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>90%</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100%</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1721</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Btn_MesLog">
       <property name="text">
        <string>MES</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_CnvLog">
       <property name="text">
        <string>CNV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_BcrLog">
       <property name="text">
        <string>BCR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_BcrNoReadLog">
       <property name="text">
        <string>BCR NOREAD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_StcLog">
       <property name="text">
        <string>STC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_JobLog">
       <property name="text">
        <string>JOB</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>1420</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>800</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="minimumSize">
        <size>
         <width>1400</width>
         <height>750</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1600</width>
         <height>750</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
	border-radius:0px;
	background-color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>300</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>500</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget{
	background-color: rgb(255, 255, 255);
	border-radius:10px;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{
	background-color: rgb(255, 255, 255);
	border-radius:10px;
}</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1750</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionhi">
   <property name="text">
    <string>hi</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
